.github/workflows/build.yaml

```diff
@@ -49,8 +49,15 @@ jobs:
           SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
         run: ./build.sh
 
+      # beta test result upload: https://docs.codecov.com/docs/test-result-ingestion-beta
+      - run: pip install codecov-cli
+      # - run: codecovcli do-upload --report-type test_results --file test/**/junit-results.xml
```

Comment:
the UI test for the mobile app only runs locally:
https://github.com/getsentry/symbol-collector/blob/4327ca596ebc4297fa01cb4051cdded1248d552d/build.sh#L35-L36